### ISSUE TITLE
cannot see replies

### ISSUE THREAD

**Issue by anarcat** [thefinn93/signal-weechat#8]
i see in the README that you have "No read receipts" - but i was assuming you'd at least see replies from other users. right now, with Signal-cli 0.5.5, i do not see replies...

i tried to enable debugging, but i do not actually see any more information.

note that i do see messages coming in from the `daemon` command output on another terminal... maybe this is related to the issues i have with automatically starting that daemon (see #3)?

**Comment by thefinn93** [comment]
yeah, if the daemon isn't running you won't see incoming messages. There are a ton of weirdnesses i've been discovering with this on systems other than mine. Debugging prints messages to a log file named whatever you set the value of the debug variable too, check that maybe?

**Comment by NickHu** [comment]
I have found this to be true for me too. It appears to be connected to the daemon fine - it linked my number correctly from weechat, and sending messages is fine, but received messages don't appear, although they are definitely received by `signal-cli` which is running in another terminal. The debug log produces no output.

**Comment by ersinakinci** [comment]
The problem is upstream with `signal-cli`: the only publishes "data messages" to DBus whereas in my testing, I've found that my Signal clients are sending "sync messages." The rest of `signal-cli` understands sync messages, it's just the part that publishes to DBus that's having a problem.

I've opened a ticket here: `https://github.com/AsamK/signal-cli/issues/207`. Unfortunately, I don't really know the Signal protocol's architecture, so I don't know how to fix it. If someone who knows the architecture could take a stab at it, that would be great (@thefinn93).
